apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/throwableAssertions.kt

```diff
@@ -38,3 +39,36 @@ fun <T : Throwable> Expect<T>.message(assertionCreator: Expect<String>.() -> Uni
  */
 fun <T : Throwable> Expect<T>.messageContains(expected: Any, vararg otherExpected: Any): Expect<T> =
     message { contains(expected, *otherExpected) }
+
+
+/**
+ * Expects that the property [Throwable.cause] of the subject *is a* [TSub] (the same type or a sub-type),
+ * creates an [Expect] of the [TSub] type for it and returns it.
+ *
+ * @return The newly created [Expect] for the property [Throwable.cause] of the subject of the assertion
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.10.0
+ */
+@Suppress("RemoveExplicitTypeArguments")
+inline fun <reified TSub : Throwable> Expect<out Throwable>.cause(): Expect<TSub> =
```

Comment:
I would prefer if you rename `TSub` to `TExpected` again. It is called `TSub` in `isA` since TSub needs to be a sub-type of `Expect<T>`. Here we don't have such a sub-type relation and thus it might be confusing.
Same same in other places